props/saber_fett263_buttons.h

```diff
@@ -2993,9 +2916,9 @@ SaberFett263Buttons() : PropBase() {}
           dial_ += direction;
           if (dial_ < 0) dial_ = NELEM(color_list_) - 1;
           if (dial_ > (int)NELEM(color_list_) - 1) dial_ = 0;
-          ShowColorStyle::SetColor(Color16(color_list_[dial_]));
+          ShowColorStyle::SetColor(Color16(color_list_[dial_].color));
 #ifdef FETT263_SAY_COLOR_LIST
-          sound_library_.SayColor(dial_);
+          sound_library_.SayColor(color_list_[dial_].color_number - 1);
```

Comment:
The -1 should be inside SayColor() I think.

Reply:
updated